helper/hex/hex.go

```diff
@@ -52,6 +52,17 @@ func EncodeUint64(i uint64) string {
 	return string(strconv.AppendUint(enc, i, 16))
 }
 
+// DecodeUint64 decodes a hex string with 0x prefix to uint64
+func DecodeUint64(hexStr string) uint64 {
+	// remove 0x suffix if found in the input string
+	cleaned := strings.Replace(hexStr, "0x", "", -1)
+
+	// base 16 for hexadecimal
+	result, _ := strconv.ParseUint(cleaned, 16, 64)
```

Comment:
Why not check error?

Reply:
My mistake we should check the error 
fixed 406fe2c54dacc8d6d08496f8a2829cc85b2e95e1 + ff727c450cd58043583531e0ac4fa9bf96d81d72
I've also added a test to confirm that encode, decode, and MustDecode works, just wasn't sure how to reproduce error. If you have an idea please help :)